docs/current_docs/tutorials/488565-openshift-gitlab.mdx

```diff
@@ -0,0 +1,110 @@
+---
+slug: /tutorials/488565/openshift-gitlab
+displayed_sidebar: "current"
+authors: ["Christian Schlatter", "Christoph Raaflaub"]
+toc_max_heading_level: 2
+date: "2023-10-27"
+---
+
+# Run Dagger on OpenShift with GitLab Runners
```

Comment:
@gerhard @kpenfound this is a straight copy of https://archive.docs.dagger.io/0.9/488564/openshift-gitlab with minor changes. Please let me know if anything stands out as needing changes.